### DISCUSSION TITLE
飞机盘无法重定向 之前好好的

### DISCUSSION THREAD

**Discussion by xiaohubig** [AlistGo/alist#6329] [Q&A]
![{22370AFF-3141-4527-B8A9-BE5E91B7B39D}(1).png](https://github.com/alist-org/alist/assets/128126510/ccec766b-87a0-4b35-9a21-8281835dd01b)

**Comment by juzipi91917** [comment]
我也遇到这个问题了

**Comment by huolanyun** [comment]
我也是 今天可能改了，获取不到链接了
